CHANGES

```diff
@@ -15,6 +15,7 @@
 - iOS: Add new MobiVM MetalANGLE backend
 - iOS: Update to MobiVM 2.3.19
 - API Addition: Added Haptics API with 4 different Input#vibrate() methods with complete Android and iOS implementations.
+- API Addition: Vector2: Added static methods for angleDeg and angleRad
```

Comment:
Does this need to be put in the right release? It looks like it will go in 1.12.0, when it should be in 1.12.2 .